Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "pistoncore-glutin_window"
-version = "0.22.0"
+version = "0.22.1"
```

Comment:
Should bump to 0.23.0 to avoid breaking existing code. See https://github.com/PistonDevelopers/piston/wiki/crates.io-and-semver#semver